binary_transparency/firmware/cmd/ft_monitor/main.go

```diff
@@ -94,6 +97,32 @@ func main() {
 			}
 
 			glog.V(1).Infof("Inclusion proof for leafhash 0x%x verified", lh)
+
+			statement := manifest.Value
+			stmt := api.FirmwareStatement{}
+			if err := json.NewDecoder(bytes.NewReader(statement)).Decode(&stmt); err != nil {
+				glog.Warningf("Fimware Statement decoding from manifest Failed reason %q", err)
```

Comment:
typo: firmware

Reply:
(also can you lowercase "Failed" while you're here?)

Reply:
Done